src/domain/feeder_actuator.py

```diff
@@ -30,6 +30,10 @@ def send_command(self, cmd):
             )
             raise Exception("Unknown command")
 
+    def actuate(self):
+        """Method to trigger the FeederActuator sending it the OPEN Command"""
+        self.send_command(self.Commands.OPEN)
```

Comment:
should we rather close the feeder whenever a detection occours? perhaps i didn't understand the command or the lable is misleading...